reana_client/cli/workflow.py

```diff
@@ -212,7 +212,17 @@ def workflow_create(ctx, file, name, skip_validation, access_token):
 
 @click.command(
     'start',
-    help='Start previously created workflow.')
+    help="""
+    Start previously created workflow
+
+    The workflow execution can be modified by setting operational
```

Comment:
```
Start previously created workflow.

The workflow execution can be further influenced by setting operational
prameters using `-p` or `--parameter` option.  The option can be repetitive.
For example, to disable caching for the Serial workflow engine, you can 
set ``-p CACHING=false``. 
 ```

P.S. BTW the value `false` is not very commonly used in the CLI world, it would be better to use 0 and 1.